### PR TITLE
Updates to documentation (README.md)

### PR DESCRIPTION
**Pull request by senthilkungumaraj**
Updates to documentation based on testing feedback including:

1. Custom vision model vs custom vision model v2 - steps to run
2. Clarification on importing pipeline using Github approach as "Import from supporting files" is removed / deprecated.

### CODE REVIEW

File: deploy/README.md [hunk 1]
```diff
@@ -126,26 +126,22 @@ Next step is to configure your resources and set them up with the required depen
...
 
 To package the Synapse pipeline, run the `package.sh` script by following the syntax below:
 
 ```bash
 ./deploy/package.sh <environmentCode> <pipelineName>
 ```
+Unzip the contents of the ZIP file and use the contents to load them to your repository.
 
-Once the above step completes, a zip file is generated. Upload the generated zip files to your Synapse Studio by following the steps below:
+**Note**: Do not bring the contents of the pipeline from [workflow folder](../src/workflow/custom-vision-model-v2) directly into your repository. You will need to run `package.sh` or `setup.sh` to replace placeholder in one or more files before checking-in the files to your repository.
```
Comment (mandarinamdar) [inline deploy/README.md:138]: Rest changes are looking good. This note is not coming out clearly from a user point of view. Need to re phrase

File: deploy/README.md [hunk 2]
```diff
@@ -304,6 +302,41 @@ To run the pipeline, open the Synapse Studio for the Synapse workspace that you
 
 - Wait for the pipeline to complete.
 
+# Running the pipeline (Custom Vision Model V2)
```
Comment (mandarinamdar) [inline deploy/README.md:305]: Do we want to repeat these contents for both pipelines ? Versioning / update at both places challenges later. Can we consolidate this section to run both the pipelines and call out deltas

**Review by mandarinamdar — CHANGES_REQUESTED** [review]

**Review by mandarinamdar — APPROVED** [review]
Based on discussions, pl address the comments. If we update the readme with those minor comment we are good to merge. Thanks

File: deploy/README.md (hunk 1 above)
Comment (senthilkungumaraj) [reply deploy/README.md]: Addressed this feedback.

File: deploy/README.md (hunk 2 above)
Comment (senthilkungumaraj) [reply deploy/README.md:312]: Discussed this section with Mandar.

**Review by sjyang18 — APPROVED** [review]
My test path is to use public image + private endpoints. Worked fine.

**Review by sushilkm — APPROVED** [review]

**Pull request merged**